StompClientLib/Classes/StompClientLib.swift

```diff
@@ -478,6 +480,17 @@ public class StompClientLib: NSObject, SRWebSocketDelegate {
         }
     }
     
+    public func stopReconnect() {
+        reconnectTimer?.invalidate()
+        reconnectTimer = nil
+    }
+    
+    public func stopReconnectAndDisconnect() {
+        reconnectTimer?.invalidate()
+        reconnectTimer = nil
+        self.disconnect()
+    }
```

Comment:
We actually do not need this function. After calling `stopReconnect`, `disconnect` function can be callable for the use-case.